docs/plugins/builtin/content-providers/title.md

```diff
@@ -27,10 +27,15 @@ The content provider doesn't support any configuration parameters.
 The content provider supports the following execution parameters:
 
 ```hcl
-content title {
-    absolute_size = <number>  # optional
-    relative_size = <number>  # optional
-    value = <string>  # required
+content "title" {
+  # Required. For example:
+  value = "some string"
+
+  # Optional. Default value:
+  absolute_size = null
```

Comment:
can you print an expected type of the attribute (int here) in the comment above?

Reply:
cty doesn't really have a notion of "int" or  "float", they are all just "number". So for now I'll display the type as "number". Is that enough?

Reply:
that's ok! If it can differentiate between numbers, strings, lists and maps - it's already good